<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysiocapAnalyseurDialogBase</class>
 <widget class="QDialog" name="PhysiocapAnalyseurDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Physiocap</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>610</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidgetHisto">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>511</width>
     <height>561</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Valeurs brutes"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tabBrutes">
    <attribute name="title">
     <string>Votre projet</string>
    </attribute>
    <widget class="QLabel" name="lab_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom du projet Physiocap (répertoire et préfixes des fichiers générés)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditProjet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDialableCharactersOnly</set>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonL93">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>L93 - EPSG 2154</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lab_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Préférence de systeme de coordonées</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonGPS">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>GPS WG84 - EPSG 4326</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonQgisProject">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Defaut de projet Qgis</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxContour">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>351</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Exclure les données brutes hors contour</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Format des cartes de points générés</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fieldComboFormats">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="minimumContentsLength">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabParametres">
    <attribute name="title">
     <string>Vos paramètres</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditDirectoryPhysiocap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonDirectoryPhysiocap">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_15">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Vecteur et champ décrivant vos contours</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditContours">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonContours">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_16">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choisir un descripteur</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fieldComboFieldContours">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="Vignoble">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>451</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="lab_32">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Interrang (cm) - Ecartement entre rangs</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxInterrangs">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>70</number>
      </property>
      <property name="maximum">
       <number>350</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>110</number>
      </property>
     </widget>
     <widget class="QLabel" name="lab_33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Intercep (cm) - Ecartement entre ceps</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxInterceps">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>70</number>
      </property>
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="lab_34">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Hauteur végétative (cm) - Haie folaire</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxHauteur">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>30</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
     </widget>
     <widget class="QLabel" name="lab_35">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Densité moyenne des sarments</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDensite">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.900000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lab_36">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cépage</string>
      </property>
     </widget>
     <widget class="QComboBox" name="fieldComboCepage">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxInfoVignoble">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Calculs détaillés :  précisez les informations du vignoble    </string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="fieldComboTaille">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lab_37">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Taille</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="Filtres_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-200</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lab_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Filtres permettant de nettoyer les données brutes</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre maximal de sarments au mètre</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMaxSarmentsParMetre_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>15</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMinDiametre_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="lab_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Tailles d'exclusion des mesures  de sarments (en mm)</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_25">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;=Diamètre de sarments retenus &lt; </string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMaxDiametre_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>90</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>28</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>18</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMinVitesse_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>25.000000000000000</double>
       </property>
       <property name="value">
        <double>1.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="lab_26">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Vitesse minimale (km/h) -  pour exclure mesure à l'arrêt</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="lab_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Répertoire des données brutes de Physiocap</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Filtres">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>451</width>
       <height>191</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="lab_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filtres permettant de nettoyer les données brutes</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre maximal de sarments au mètre</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMaxSarmentsParMetre">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>25</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMinDiametre">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="lab_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tailles d'exclusion des mesures  de sarments (en mm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_20">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;=Diamètre de sarments retenus &lt; </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxMaxDiametre">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>6</number>
      </property>
      <property name="maximum">
       <number>28</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>28</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMinVitesse">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>25.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lab_18">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Vitesse minimale (km/h) -  pour exclure les mesures à l'arrêt</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Histogrammes</string>
    </attribute>
    <widget class="QLabel" name="lab_38">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Non implémenté dans cette version</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Version 1.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PhysiocapAnalyseurDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PhysiocapAnalyseurDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
